src/components/Layout/NavigationLayout/GenreNavigation/GenreNavigation.tsx

```diff
@@ -1,11 +1,51 @@
+import { GENRE_FILTER } from '../../../../constants/constants';
+import { moviesActions } from '../../../../redux/moviesSlice';
+import { useAppDispatch, useAppSelector } from '../../../../redux/store';
+
 import './GenreNavigation.scss';
 
 const GenreNavigation = (): JSX.Element => {
-  const genres = ['All', 'documentary', 'comedy', 'horror', 'crime'];
+  const filters = useAppSelector((state) => state.movies.filters);
+  const dispatch = useAppDispatch();
+  const allFilters = GENRE_FILTER[0].id;
```

Comment:
This variable could me moved outside the component, since it is a constant value.

Reply:
make sense. Thanks